essential-projects/routing_contracts
# Split content of interface.ts

## What did you change?

-All interfaces are now located in separate files.

## How can others test the changes?

-Look through the source folder, you'll immediately see the changes.
-Build a solution that uses the metadata_contracts project. The build should run exactly as before, since all changes made were exclusively cosmetic and non-functional.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
